setup.cfg

```diff
@@ -7,6 +7,7 @@ version = 0.1.dev0
 install_requires=
     numpy
     statistics
+    astropy.table
```

Comment:
> "The library must not depend on pandas, dask, xarray, or any other library implementing a similar functionality."

astropy.table is very similar to pandas, so implementing functionality on top of it is not permitted.